R50_fwd_pass.py

```diff
@@ -0,0 +1,174 @@
+import torch
+from torch.cuda.amp import autocast
+from torchvision import transforms, datasets
+
+import numpy as np
+import argparse
+import copy
+
+from imagenetv2_pytorch import ImageNetV2Dataset
+from utils.model import *
+
+batch_size = 2048
+img_size = 224
+nesting_start=3
```

Comment:
Consistent spacing "=" should have space on both sides if it is not an argument. 

Also, the magic numbers can be macros first.